### PR TITLE
add leetcode.signinByCookie but simple change

### PR DESCRIPTION
**Pull request by yihong0618**
This is very simple change maybe for  short-term  solution for #478 
Sorry for the first time write ts.
Please help me review~
@jdneo 
Thanks a lot.

### CODE REVIEW

**Comment by jdneo** [comment]
Is it possible to leverage the cookie extension:
 - [cookie.chrome](https://github.com/leetcode-tools/leetcode-cli-plugins/blob/master/plugins/cookie.chrome.js)
 - [cookie.firefox](https://github.com/leetcode-tools/leetcode-cli-plugins/blob/master/plugins/cookie.firefox.js)
to bypass the reCAPTCHA?

**Comment by yihong0618** [comment]
Yes, but I think this way is not so simple, I will try it tomorrow.

**Comment by jdneo** [comment]
@yihong0618 I just took a look about the document. The cookie plugins need more requirements and they are different for different platforms. So I think it's not an ideal solution. Just forget it.

**Comment by yihong0618** [comment]
@jdneo me either. And this solution is for one-time session at that time.

**Comment by yihong0618** [comment]
@jdneo 
cookie.chrome work.
But I think its not a good idea, it is no essential difference from the paste cookie method

File: README.md [hunk 1]
```diff
@@ -40,8 +40,9 @@
 
 - Simply click `Sign in to LeetCode` in the `LeetCode Explorer` will let you **sign in** with your LeetCode account.
 
-- You can also use the following command to sign in/out:
+- You can also use the following command to sign in/cookie in/out:
   - **LeetCode: Sign in**
+  - **LeetCode: Cookie in**
```
Comment (jdneo) [inline README.md]: How about renaming it to `Sign in (by cookie)`

Comment (yihong0618) [reply README.md]: Of course, should I also change name in **leetcode.cookieIn command** ?

Comment (yihong0618) [reply README.md]: @jdneo 
Have changed it~

File: package.json [hunk 2]
```diff
@@ -38,11 +38,17 @@
...
         "onCommand:leetcode.switchDefaultLanguage",
+        "onCommand:leetcode.signinByCookie",
         "onView:leetCodeExplorer"
     ],
     "main": "./out/src/extension",
     "contributes": {
         "commands": [
+            {
+                "command": "leetcode.signinByCookie",
+                "title": "Sign In By Cookie",
```
Comment (jdneo) [inline package.json]: nit: `By` should be lower case since it's a preposition

**Comment by jdneo** [comment]
Hi @yihong0618, almost there. Just a typo.

File: package.json (hunk 2 above)
Comment (yihong0618) [reply package.json]: Ok, I will change it right now

Comment (yihong0618) [reply package.json]: > "command": "leetcode.signinByCookie", ? 
This "by" also?

**Comment by yihong0618** [comment]
Hi I fixed the typo, is that right?

**Comment by jdneo** [comment]
LGTM.  Thank you @yihong0618 for your contribution! Well done!

**Comment by jdneo** [comment]
Will release this fix before the end of this week

**Comment by yihong0618** [comment]
Thank you~

**Pull request merged**